src/docs/index.md

```diff
@@ -2,26 +2,27 @@
+By establishing a common framework, SSSOM enhances the consistency, quality, and discoverability of mappings, thereby supporting more effective data integration and analysis.
 
-1. a machine-readable and extensible vocabulary to describe metadata of mappings;
-2. a data model to represent mappings and their associated metadata;
-3. several file formats to represent sets of mappings on disk and on the network.
+- **Standardization**: SSSOM provides a unified format for representing semantic, or ontological, mappings, making it easier for different systems and organizations to exchange mapping data consistently.
+- **Interoperability**: By using SSSOM, data from diverse sources can be integrated more seamlessly, allowing for improved data analysis and research across various fields, including biology, healthcare, and information technology.
 
-Beyond defining the standard itself, the **SSSOM Core Team** also aims to implement open and community-driven collaborative workflows designed to evolve the standard continuously to address changing requirements and mapping practices, and to provide reference tools and software libraries for working with the standard.
+Beyond defining the standard itself, the **SSSOM Core Team** and the SSSOM community also develop reference tools and software libraries for working with the standard.
 
-## SSSOM at a glance
+## SSSOM at a glance: Model and Exchange Format
 
-### Basic concepts
+### Basic model
 
 The [data model](spec-model.md) of SSSOM is centered around two fundamental concepts: mappings and mapping sets.
 
-A **SSSOM mapping** is a statement that there is a correspondence of some sort between two semantic entities. It comprises two components:
+A **SSSOM mapping** is a statement that there is a correspondence between two semantic entities. It comprises two components:
 
 1. The **core mapping** (or **raw mapping**), which is a triple `<subject, predicate, object>` that represents the correspondence itself between a subject entity, for example a class in an ontology, and an object entity, for example an identifier in some database, via a semantic mapping predicate, for example `skos:exactMatch`.
-2. **Metadata** that provide supplementary pieces of information about the core mapping. This notably includes information pertaining to the *provenance* of the statement (for example, who emitted the statement, in other words who decided that the subject and the object should be mapped) and its *justification* (why should the subject and the object be mapped).
+2. **Metadata** that provide supplementary pieces of information about the core mapping. This notably includes information about the *provenance* of the statement (for example, who authored the statement) and its *justification* (a reason that supports the fidelity subject and the object, such as expert review, or exact lexical matching on the entities' primary names).
```

Comment:
I think there's a word or two missing after "fidelity", but I'm not sure whether it should say "fidelity **_of the_** subject and the object" or "fidelity **_between the_** subject and the object" or something else along that line.

Reply:
```suggestion
2. **Metadata** that provide supplementary pieces of information about the core mapping. This notably includes information about the *provenance* of the statement (for example, who authored the statement), the *confidence* with which the mappings holds, and its *justification* (a reason that supports the fidelity of the mapping between the subject and the object, such as _expert review_, or _exact lexical matching_ on the entities' primary names).
```